packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -38,7 +38,7 @@ export class Pragma extends Op {
     if (this.line > 1) {
       throw new RuntimeError(RUNTIME_ERRORS.TEAL.PRAGMA_NOT_AT_FIRST_LINE, { line: line });
     }
-    if (args[0] === "version" && (args[1] === '1' || args[1] === '2')) {
+    if (args[0] === "version" && Number(args[1]) <= MaxLogicSigVersion) {
```

Comment:
How about renaming `MaxLogicSigVersion` to `MaxTEALVersion`?

Reply:
makes sense.